UKHomeOffice/cerberus-service
# COP-10790: Create tasks with co-traveller block empty

## Description

Tests added to create tasks with Co-travellers block empty and check the task details page.

## To Test

npm run cypress:runner

run the following tests from `create-tasks.sepc.js`

`Should create a task with a payload contains RoRo Accompanied - Empty Co-passenger array`
`Should create a task with a payload contains RoRo Tourist - Empty Co-passenger array`

## Developer Checklist

\* Required
